tests/test-sources/plugins/by-name/aw-watcher/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  empty = {
+    # don't run tests as they try to access the network.
+    test.runNvim = false;
+    plugins.aw-watcher.enable = true;
+  };
+
+  defaults = {
+    # don't run tests as they try to access the network.
+    test.runNvim = false;
+    plugins.aw-watcher = {
+      enable = true;
+
+      settings = {
+        bucket = {
+          hostname = "nixvim";
+          name = "aw-watcher-neovim_nixvim";
+        };
+        aw_server = {
+          host = "127.0.0.1";
+          port = 5600;
+          ssl_enable = false;
+          pulsetime = 30;
+        };
+      };
+    };
+  };
```

Comment:
Thanks!

It would be great to also include an `example` test case with the content of `settingsExample` from the plugin module.